src/utils/vaultDbProvider.ts

```diff
@@ -67,66 +65,23 @@ export class SecretsDbProvider implements IDbProvider {
       const amtpass = data.data.AMT_PASSWORD
       return [user, amtpass]
     } catch (error) {
-      this.logger.error('Error while retrieving device credentials\r\n')
-      this.logger.error(error)
+      this.logger.error('Error while retrieving device credentials :', error)
       return null
     }
   }
 
   async IsGUIDApproved (guid: string, cb: any): Promise<void> {
```

Comment:
i'd like to start to "generazing" the vaultDbProvider a bit, so, in essence should the function simple be "GetDevice Secret()" ?